lib/src/components/VernaForm/index.spec.tsx

```diff
@@ -19,29 +21,51 @@ describe('VernaForm', () => {
     await userEvent.click(_.last(buttons)!);
   }
 
-  it('should render a basic form', async () => {
+  const Wrapper = ({
+    isEditor,
+    selector,
+    uiSchema,
+    widgets,
+  }: {
+    selector?: string;
+    isEditor?: boolean;
+    uiSchema?: UiSchema;
+    widgets?: WidgetsType;
+  }) => {
     const translations = translationsFactory();
-
-    render(
-      <VernaProvider
-        isEditor
-        defaultSchema={schemaFactory()}
-        defaultUiSchema={uiSchemaFactory()}
-        defaultWidgets={widgetsFactory()}
-        locale="en"
-        translations={translations}
-      >
-        <VernaForm />
-      </VernaProvider>,
+    return (
+      <Suspense fallback="Loading...">
+        <VernaProvider
+          defaultSchema={schemaFactory()}
+          defaultSelector={selector}
+          defaultUiSchema={uiSchema}
+          defaultWidgets={widgets}
+          isEditor={isEditor}
+          translations={translations}
+        >
+          <div data-testid="wrapper">
+            <VernaForm />
+          </div>
+        </VernaProvider>
+      </Suspense>
     );
+  };
+
+  it('should render a basic form', async () => {
+    render(<Wrapper uiSchema={uiSchemaFactory()} widgets={widgetsFactory()} />);
+    // Wait that the form is rendered...
+    await waitFor(() => {
+      expect(screen.queryByTestId('wrapper')).toBeInTheDocument();
+    });
+
     // - A fieldset legend should be displayed with form title
-    screen.getByRole('group', { name: translations.en.root_title });
+    screen.getByRole('group', { name: 'A registration form' });
 
     // - A paragraph with form description should be displayed
-    screen.getByText(translations.en.root_description);
+    screen.getByText('Registration form description');
 
     // - The first section should be displayed
-    screen.getByRole('group', { name: translations.en.root_testSection_title });
+    screen.getByRole('group', { name: 'First section' });
```

Comment:
Why use translations object and hardcode translations there?
I thought it was to avoid diff in translations files but it does the opposite there doesn't it?